src/Appwrite/Platform/Workers/Messaging.php

```diff
@@ -184,8 +185,24 @@ private function processMessage(Database $dbForProject, Document $message): void
                         };
 
                         try {
-                            $adapter->send($data);
-                            $deliveredTotal += \count($batch);
+                            $response = new Response($provider->getAttribute('type'));
+                            $response->fromArray(\json_decode($adapter->send($data)));
```

Comment:
Can `send` just return `Response` instead of doing it this way?

Reply:
we can change this once we have all the providers unified, for now we can return string.